packages/atlas/src/views/studio/YppDashboard/YppDashboard.config.tsx

```diff
@@ -1,21 +1,32 @@
 import { SvgTierIcon1, SvgTierIcon2, SvgTierIcon3 } from '@/assets/icons'
 import { Text } from '@/components/Text'
+import { atlasConfig } from '@/config'
 import { absoluteRoutes } from '@/config/routes'
+import { formatNumber } from '@/utils/number'
 
-export const TIERS = [
-  {
-    rules: '<5K subscribers',
-    icon: <SvgTierIcon1 />,
-  },
-  {
-    rules: '5K-25K subscribers',
-    icon: <SvgTierIcon2 />,
-  },
-  {
-    rules: '>25K subscribers',
-    icon: <SvgTierIcon3 />,
-  },
-]
+const configTiers = atlasConfig.features.ypp.tiersDefinition?.tiers
+
+export const TIERS = configTiers
+  ? [
+      {
+        rules: `<${formatNumber(configTiers?.[1].minimumSubscribers - 1)} subscribers`,
```

Comment:
I don't think that `-1` is needed, we want the first tier to be `<100` and not <99`, since 99 is still first tier